packages/react/src/internal/menuButton/MenuButton.module.scss

```diff
@@ -14,6 +14,7 @@
   --menu-item-color-highlighted: var(--color-white);
 
   border-radius: 0;
+  gap: var(--spacing-2-xs);
   position: relative;
```

Comment:
In Chrome: "The display: block property prevents gap from having an effect.
Try setting display to something other than block."

Reply:
Need to have a look at this